core/templates/xregistry_openapi_template.json

```diff
@@ -1,1068 +1,1937 @@
+                            "format": "int64"
+                        }
+                    },
+                    {
+                        "$ref": "#/components/parameters/inline"
+                    },
+                    {
+                        "$ref": "#/components/parameters/filter"
+                    },
+                    {
+                        "$ref": "#/components/parameters/doc"
+                    },
+                    {
+                        "$ref": "#/components/parameters/specversion"
+                    }
+                ],
+                "requestBody": {
+                    "description": "The resource document or xRegistry metadata (depending on URL suffix $details).",
+                    "content": {
+                        "application/octet-stream": {
+                            "schema": {
+                                "type": "string",
+                                "format": "binary"
+                            }
+                        },
+                        "application/json": {
+                            "schema": { "$ref": "{%-resourceTypeReference-%}" }
+                        }
+                    }
+                },
+                "responses": {
+                    "200": {
+                        "description": "The updated resource.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "{%-resourceTypeReference-%}"
+                                }
+                            }
+                        }
+                    },
+                    "400": {
+                        "$ref": "#/components/responses/BadRequest"
+                    },
+                    "401": {
+                        "$ref": "#/components/responses/Unauthorized"
+                    },
+                    "404": {
+                        "$ref": "#/components/responses/NotFound"
+                    },
+                    "409": {
+                        "$ref": "#/components/responses/Conflict"
+                    },
+                    "500": {
+                        "$ref": "#/components/responses/InternalServerError"
+                    }
+                }
+            },
+            "post": {
+                "summary": "Post new resource version",
+                "description": "Creates a new version of the resource. If the resource does not exist, it is created at version 1; if it exists, the new version is created (default version may be updated using setdefaultversionid). (See spec: Creating or Updating Resources and Versions, Default Version of a Resource)",
+                "operationId": "post{%-resourceNameSingular-%}",
+                "tags": ["Resource"],
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/resource-description"
```

Comment:
I assume these are headers - if so, we should include all of the scalar attributes that can appear on a resource/version

Reply:
need to deal with hasdoc=false